projects/lf-documentation/src/app/getting-started-cdn-example/getting-started-cdn-example.component.html

```diff
@@ -0,0 +1,163 @@
+<div class="toc-container">
+  <p class="toc-container-header"> Contents</p>
+  <ul>
+    <li>
+      <a href="#cdn-example-angular">Example: <code>lf-checklist</code> In an Angular Project Via CDN</a>
```

Comment:
You don't need to put `Via CDN` in all the titles since this page is all CDN

There could be a title though
`Examples using the CDN`